examples/v2/config_chat2.nim

```diff
@@ -248,7 +248,42 @@ type
       desc: "the pubsub topic for which rln-relay gets enabled",
       defaultValue: "/waku/2/default-waku/proto"
       name: "rln-relay-pubsub-topic" }: string
-
+      
+    rlnRelayDynamic* {.
+      desc: "Enable  waku-rln-relay with on-chain dynamic group management: true|false",
```

Comment:
```suggestion
      desc: "Enable waku-rln-relay with on-chain dynamic group management: true|false",
```

Reply:
done in 604add4b381d7a1f360f3165f084b0eff93509c7